pcomfortcloud/authentication.py

```diff
@@ -338,9 +364,7 @@ def _get_header_for_api_calls(self, no_client_id=False):
             "x-app-timestamp": timestamp,
             "x-app-type": "1",
             "x-app-version": self._app_version,
-            # Seems to work by either setting X-CFC-API-KEY to 0 or to a 128-long hex string
-            # "X-CFC-API-KEY": "0",
-            "x-cfc-api-key": generate_random_string_hex(128),
+            "x-cfc-api-key": self._get_api_key(timestamp, self._token["access_token"]),
```

Comment:
I struggled a long time with this one, but the token sent into _get_api_key is wrong. After fixing that it's working for me.

```suggestion
            "x-cfc-api-key": self._get_api_key(timestamp, token_response["access_token"]),
```

Reply:
Yeah, this shouldn't be changed